tests/nnstreamer_filter_python/runTest.sh

```diff
@@ -47,13 +47,13 @@ popd
 
 # Passthrough test
 export PYTHONPATH=../../build/ext/nnstreamer/tensor_filter/:$PYTHONPATH
-PATH_TO_SCRIPT="python/passthrough.py"
+PATH_TO_SCRIPT="../test_models/models/passthrough.py"
```

Comment:
I think it's more like `data` rather than `models`.

Reply:
But this is passed a model file to tensor_filter as it operates on the data passed to tensor_filter.

Reply:
Yes... I reused the existing model parameter to support this. Then, it's fine.

Reply:
But... it was related to the internal implementation.
I think it's still weird if these example python custom filter scripts are co-located with `*.pb` in `/models/` folder.

Reply:
I think these are models as they provide the specification to take in data, process them and give output, just like .pb file. The only difference is pb file needs a framework like tensorflow to understand them but these .py files are self sufficient.
They much closer to models than data.

Reply:
Yes, makes sence